docs/tools/developer-tools/lab/quickstart-with-lab.mdx

```diff
@@ -0,0 +1,31 @@
+# Using Lab with Quickstart
+
+## Overview
+
+[Quickstart](https://github.com/stellar/quickstart) provides an easy way to set up a local Stellar network environment. Specifically, Quickstart docker image bundles Stellar Core with Horizon, RPC, Friendbot, and the necessary PostgreSQL databases. Now it is possible to use Stellar Lab as an interface on Quickstart.
+
+## Prerequisites
+
+- Stellar CLI
+- Docker
+
+## Start Stellar Quickstart
+
+For `testnet`, start Quickstsart with the Stellar CLI using the following command:
+
+```sh
+stellar network start testnet
```

Comment:
we should use `stellar container start` as recommended [here](https://developers.stellar.org/docs/tools/developer-tools/cli/stellar-cli#stellar-network-start)

Reply:
actually i think it's `stellar network container start` https://developers.stellar.org/docs/tools/developer-tools/cli/stellar-cli#stellar-network-container